### PR TITLE
React capstone project - Metrics webapp

### PR DESCRIPTION
**Pull request by GOlivierNation**
### What is Done in this PR:
*******************************
API First you need to choose an API to base the development of the webapp on. The API should allow you to get numeric values based on a parameter. We recommend that you choose an API that is open or needs an API key. If you choose an API that require authentication, you should implement it on your own.

Some example APIs are:

[Financial modeling prep API](https://financialmodelingprep.com/developer/docs/): stock data.
[Air pollution API](https://openweathermap.org/api/air-pollution): air quality data.
You can find many APIs in [this GitHub repo](https://github.com/public-apis/public-apis) or in [ProgrammableWeb](https://www.programmableweb.com/category/all/apis). Some of the APIs requires a token for authentication and some others are just open.

Design

You should follow these [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), including:
Colors (select one main color for your website).
Typography: font face, size and weight.
Layout: composition and space between elements.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,180 @@
...
+This project is build with:
+
+The page will reload if you make edits.\
+You will also see any lint errors in the console.
+- ![HTML5](https://img.shields.io/badge/-HTML5-000000?style=flat&logo=html5&logoColor=ffffff&labelColor=E34F26) ![CSS3](https://img.shields.io/badge/-CSS3-000000?style=flat&logo=css3&logoColor=ffffff&labelColor=1572B6)
+- ![React](https://img.shields.io/badge/-React-000000?style=flat&logo=react)  ![Redux](https://img.shields.io/badge/-Redux-000000?style=flat&logo=redux&logoColor=ffffff&labelColor=violet)
+
+## Live Demo Link
+
+### [Live Demo](#)
```
Comment (VICTOR-CHUKWU) [inline README.md]: - [ ] Kindly add a live link of your website

File: README.md [hunk 2]
```diff
@@ -1,2 +1,180 @@
...
+
+## Loom Video Link
+
+### [Loom](#)
```
Comment (VICTOR-CHUKWU) [inline README.md]: - [ ] Kindly add a demo video link not more than 5 mins describing the features of your project

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @GOlivierNation , 

## STATUS: REQUIRED CHANGES :recycle: 
Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md [hunk 3]
```diff
@@ -1,2 +1,180 @@
...
+- ![HTML5](https://img.shields.io/badge/-HTML5-000000?style=flat&logo=html5&logoColor=ffffff&labelColor=E34F26) ![CSS3](https://img.shields.io/badge/-CSS3-000000?style=flat&logo=css3&logoColor=ffffff&labelColor=1572B6)
+- ![React](https://img.shields.io/badge/-React-000000?style=flat&logo=react)  ![Redux](https://img.shields.io/badge/-Redux-000000?style=flat&logo=redux&logoColor=ffffff&labelColor=violet)
+
+## Live Demo Link
+
+### [Live Demo](https://metricdatacovid19.netlify.app/)
+
+## Loom Video Link
+
+### [Loom](https://www.loom.com/share/cdba46e67d90463a8dcfe7858b2cd7b5)
```
Comment (DammyShittu) [inline README.md]: - [ ] Your video has no audio. Kindly redo the video with sound presenting the project and remember to show the piece of code you find interesting or are proud of making.

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @GOlivierNation,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lameck1 — APPROVED** [review]
Hi, @GOlivierNation,

You've done well working on the concerns from the previous review 👍🏼.
The tasks for this milestone are now successfully implemented 💯.

## Highlights
* Correct gitflow practice ✔️ 
* Good commit messages ✔️ 
* Proffessional README ✔️ 
* App deployed and available online ✔️ 
* In the project:
   * Good usage of the react-router-dom package ✔️ 
   * Clean codebase ✔️ 
   * Redux store and redux logic properly implemented ✔️  
   * All expected features of the MVP functioning as expected ✔️

All the best with the next project.

You are free to merge this branch :shipit:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**Please, remember to tag me in your question so I can receive the notification.**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**